content/html/concepts/attributes/terms/src.md

```diff
@@ -0,0 +1,48 @@
+﻿---
+Title: 'src'
+Description: 'An HTML attribute used to specify the location of a digital resource.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Attributes'
+  - 'Elements'
+  - 'Style'
+CatalogContent: 
+   - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The  **`src`** attribute is  used to specify the location of a digital source, which is typically a URL. It is required for HTML elements like [`<img>`](https://www.codecademy.com/resources/docs/html/images) so that they can appear on a website or application. 
+
+## Syntax
+
+```pseudo
+<element  src="url">
+```
+
+
+`src` can also be used in the following elements :
```

Comment:
```suggestion
`src` can be used in the following elements :
```

I'd include `<img>` in the table.

Reply:
Thanks for the suggestion @SSwiniarski, but I believe adding the image element to the chart will create repetition since I already mentioned that the src attribute is required for images in the first paragraph. I created the chart to point out other HTML elements that use src.

Reply:
@CBID2 I understand that, but I think it's more important that the table is complete since it is the major part of this entry.

Reply:
Ok